### PR TITLE
Add groupings

### LINKED ISSUE #22: Groupings
Grouping expressions should be supported `(` and `)`

### PR DESCRIPTION
**Pull request by onbjerg**
Closes #22

### CODE REVIEW

File: test/examples/examples.js [hunk 1]
```diff
@@ -43,6 +45,7 @@ const cases = [
 test('Examples', async (t) => {
   for (let [input, expected] of cases) {
     const actual = await evaluateRaw(input.source, input.bindings)
+    console.log(actual)
```
Comment (izqui) [inline test/examples/examples.js]: remove log?

**Comment by izqui** [comment]
Looks like linter is failing tho

File: test/examples/examples.js (hunk 1 above)
Comment (onbjerg) [reply test/examples/examples.js]: whoopsies

**Comment by onbjerg** [comment]
I'm just going to ignore Coveralls for now

**Pull request merged**